### PR TITLE
Deprecate the `cloud`, `cloud-aws` and `cloud-azure` packages

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Part of: https://github.com/pulumi/pulumi-cloud/issues/737

### CODE REVIEW

File: api/Makefile [hunk 1]
```diff
@@ -12,4 +12,3 @@ build::
 	cp ../README.md ../LICENSE bin/
 
 lint::
-	tslint -c ../tslint.json -p tsconfig.json
```
Comment (pgavlin) [inline api/Makefile]: Does the linter complain about all the deprecation?

Comment (CyrusNajmabadi) [reply api/Makefile]: it's more that i overwrote existing comments, which had linting-preferences specified in them.  i don't want to have to go add those things back in.  so just easier (imo) to disable linting entirely.

Comment (pgavlin) [reply api/Makefile]: Ahh, makes sense. I think that should be fine given the plans here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## Deprecated (July 15, 2019)
+
+- `@pulumi/cloud` before is deprecated.  Existing `@pulumi/cloud` packages (up to v0.18.1) will
```
Comment (pgavlin) [inline CHANGELOG.md]: `before` what?

Comment (CyrusNajmabadi) [reply CHANGELOG.md]: oops.

**Comment by CyrusNajmabadi** [comment]
Closint out.  This is *not* the strategy we will take with cloud.  Instead, we will shift all our messaging and display around this package to indicate that it is a *preview*.

**Pull request closed without merging**